.github/workflows/build_and_test_lua.yml

```diff
@@ -31,9 +31,9 @@ jobs:
         sudo apt install -y libzstd-dev liblz4-dev libsnappy-dev
 
     - name: configure cmake build for tidesdb
-      run: | 
+      run: |
         cd tidesdb
-        cmake --debug-output  -S . -B build && make -C build/
+        cmake -DTIDESDB_WITH_SANITIZER=OFF --debug-output  -S . -B build && make -C build/
```

Comment:
@guycipher  this is how we should pass arg to cmake so it won't build libr with sanitizers then no dirty hacks needed